frontend/javascripts/oxalis/view/left-border-tabs/layer_settings_tab.tsx

```diff
@@ -976,6 +988,8 @@ class DatasetSettings extends React.PureComponent<DatasetSettingsProps, State> {
       );
 
     const defaultLayerViewConfig = getDefaultLayerViewConfiguration();
+    const isHistogramAvailable =
+      isHistogramSupported(elementClass) && layerName != null && isColorLayer;
```

Comment:
I know, it's merely moved, but still: is layerName not always not-null? I think that part can be removed from the condition.